### PR TITLE
Extensively clarify "Identify IP address" section.

### PR DESCRIPTION
**Pull request by chjfth**
Previous version text is too simplified. 

Previous version erroneously said `hostname -i`. It should be `hostname -I` actually.

### CODE REVIEW

**Comment by chjfth** [comment]
@microsoft-github-policy-service agree

File: WSL/basic-commands.md [hunk 1]
```diff
@@ -168,8 +168,44 @@ To terminate the specified distribution, or stop it from running, replace `<Dist
 
 ## Identify IP address
 
-- `wsl hostname -i` for the IP address of your Linux distribution installed via WSL 2 (the WSL 2 VM address)
-- `cat /etc/resolv.conf` for the IP address of the Windows machine as seen from WSL 2 (the WSL 2 VM)
```
Comment (craigloewen-msft) [inline WSL/basic-commands.md:172]: We should use this command instead:
`ip route show | grep -i default | awk '{ print $3}'`

see here: https://learn.microsoft.com/en-us/windows/wsl/networking#ipv6-access

File: WSL/basic-commands.md [hunk 2]
```diff
@@ -168,8 +168,44 @@ To terminate the specified distribution, or stop it from running, replace `<Dist
 
 ## Identify IP address
 
-- `wsl hostname -i` for the IP address of your Linux distribution installed via WSL 2 (the WSL 2 VM address)
```
Comment (craigloewen-msft) [inline WSL/basic-commands.md:171]: This should be kept simple as just a quick two line explanation, and we should link out deeper info to the networking page.

**Review by craigloewen-msft — DISMISSED** [review]

**Comment by mattwojo** [comment]
Thanks for this contribution @chjfth! Merging!

**Pull request merged**